apps/webbsite/src/styles/globals.css

```diff
@@ -57,67 +54,169 @@
   @apply !bg-mono-200;
 }
 
-/** Headings and subheadings */
-
+/* Headings, sub-headings, card texts */
 .heading-1 {
-  @apply text-[32px] leading-[46px] tracking-tight font-bold text-mono-200 font-satoshi
-    md:text-[84px] md:leading-[96px] md:tracking-normal;
+  @apply text-base
+  text-[34px] leading-[45.9px] tracking-[-0.011em] /** small screen */
```

Comment:
You don't need the `text-base` here because you already set the font size by `text-[34px]` and `leading-[45.9px]`. All the `text-base` classes below can be removed.